agreement/types.go

```diff
@@ -68,18 +68,18 @@ const (
 )
 
 func (s step) nextVoteRanges() (lower, upper time.Duration) {
-	extra := recoveryExtraTimeout // eg  2500 ms
-	lower = deadlineTimeout       // eg 17500 ms (15000 + 2500)
-	upper = lower + extra         // eg 20000 ms
+	extra := recoveryExtraTimeout // eg  2000 ms
+	lower = deadlineTimeout       // eg 17000 ms (15000 + 2000)
+	upper = lower + extra         // eg 19000 ms
 
 	for i := next; i < s; i++ {
 		extra *= 2
 		lower = upper
 		upper = lower + extra
 	}
 
-	// e.g. if s == 14
-	// extra = 2 ^ 8 * 2500ms = 256 * 2.5 = 512 + 128 = 640s
```

Comment:
where 2500ms or 2000ms come from?

Reply:
small lambda = 2000ms, while the comment said it is 2500ms, so it is wrong.

`extra` started from 2000 ms, so I need to fix the computation in comments too...